core/src/main/kotlin/knote/poet/PageDependency.kt

```diff
@@ -44,15 +44,15 @@ object PageDependency : KLogging() {
         return output.resolve("$fileName.kt")
     }
 
-    fun <T> injectFromNotebook(notebookId: String, pageId: String): PageResult<Nothing?, T> =
-        object : PageResult<Nothing?, T> {
+    fun <T> injectFromNotebook(notebookId: String, pageId: String): DelegatedResult<Nothing?, T> =
+        object : DelegatedResult<Nothing?, T> {
```

Comment:
Man, Kotlin sure makes this look messy. nothing to do with this here this is just something that could probably be improved on Kotlin's side haha